examples/oidc/README.md

```diff
@@ -0,0 +1,119 @@
+# Secure a service with OIDC authentication
+
+> Note to try this out with minikube, make sure you enable RBAC correctly as explained [here](../minikube-rbac).
+
+This example ilustrates how to use `kube-rbac-proxy` with OIDC support to secure k8s apiserver.
```

Comment:
ahh, ... now once I reached this sentence I recognize that securing the API server is indeed your canonical use case here ;-)

also pinging @brancz: is securing the API server itself, using the kube-rbac-proxy, really something that is in scope for this project? I believe, it is not, but I may be mistaken here.

Independent, it would be great to have an example, similar to i.e. the `non-resource-url` one, where an arbitrary Pod is being secured using this proxy.